content/src/service/synchronization/batchDeployer.ts

```diff
@@ -106,7 +106,6 @@ export function createBatchDeployerComponent(
               DeploymentContext.SYNCED
             )
 
-            components.deployedEntitiesFilter.add(entity.entityId)
```

Comment:
I think this was necessary. Was it moved somewhere else?

Reply:
tl;dr: it's called twice
The `batchDeployer` calls `deployEntityFromRemoteServer` just the line above, which then calls `deployDownloadedEntity` that finally calls `deployEntity` where the `add` is done. This seems good to me because there's only one point where the component is mutated.